### DISCUSSION TITLE
In which country does the Tango 🕺💃 originate?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#659] [Galaxy Trivia] [answered]
**Bonus:** Which country also claims its origin?

**Comment by peterbaccaro** [comment, marked as answer]
**Argentina** and **Uruguay**.

The Tango is a dance and musical genre that originated in the Rio de la Plata region, which includes Buenos Aires, Argentina, and Montevideo, Uruguay. The exact origins of the Tango are not entirely clear, as the dance evolved over time and was influenced by various cultures and musical styles. However, it is generally believed that the Tango emerged in the late 19th century in the working-class neighborhoods of Buenos Aires, where immigrants from Europe and Africa came together and blended their cultural traditions. Today, the Tango is a popular dance that is enjoyed around the world and is considered an important part of the cultural heritage of Argentina and Uruguay.
